docs/bir-spec/src/test/java/org/ballerinalang/birspec/BIRTestUtils.java

```diff
@@ -590,4 +665,18 @@ Bir getActualBIR() {
             return actualBIR;
         }
     }
+
+    static class ExpectedScopeEntry {
+        public int id;
+        public int instructionOffset;
+        public int hasParent;
+        public Integer parentId;
```

Comment:
shall we have getter setter instead of having public variables? If it is a Immutable class let's make them final

Reply:
This is used as a data class without any functionality. Using getters setters would be too much imo. I'll make these final instead.